.vs/ProjectSettings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
To be honest, I have no idea. This one does not have any sensible contents, but is governed by git. I did not touch the .gitignore when I created the fork. Since they are inside the .vs folder I do not really understand why they are include in the pull request. I will try to get rid of them, but I do not yet know how to do that.